src/Microsoft.ML.Core/Data/DateTime.cs

```diff
@@ -81,13 +78,13 @@ public bool IsNA
             get
             {
                 AssertValid();
-                return (ulong)_ticks.RawValue > MaxTicks;
+                return (ulong)_ticks > MaxTicks;
             }
         }
 
         public static DvDateTime NA
         {
-            get { return new DvDateTime(DvInt8.NA); }
+            get { return new DvDateTime(long.MinValue); }
```

Comment:
>get { return new DvDateTime(long.MinValue); } [](start = 12, length = 45)

So long as you're changing this anyway may as well use simpler C# 7 syntax.

Reply:
Sure but you are aware this file will be deleted in the DateTime PR, otherwise that PR won't be approved or checked-in...

---
In reply to: [212033279](https://github.com/dotnet/machinelearning/pull/683#discussion_r212033279) [](ancestors = 212033279)